### PR TITLE
Improve cryo tubes

### PR DESCRIPTION
**Pull request by Generalcamo**
This PR greatly improves cryo tubes by porting over the improvements made on them from Baystation over the past several years. This includes:

- A significant buff to cryoxadone and clonexadone (however, this also comes with the removal of innate healing of the cryotubes, making these chemicals mandatory)
- New modes to prioritize stasis or metabolism, for ease of use.
- New sprites to Cryo tubes from baystation, including sprites for bad and warning temperature thresholds.

In addition, the Cryo tube interface has been ported to TGUI.

### Asset Licenses
The following assets that **have not** been created by myself are included in this PR:

| Path | Original Author | License |
| --- | --- | --- |
| icons/obj/cryogenics_split.dmi | Remission & cuddleandtea (Baystation 12) | CC-BY-SA 3.0 |

### CODE REVIEW

**Comment by Generalcamo** [comment]
!review

File: code/game/machinery/cryo.dm [hunk 1]
```diff
@@ -76,113 +109,136 @@
...
-		occupantData["stat"] = occupant.stat
-		occupantData["bodyTemperature"] = occupant.bodytemperature
-		occupantData["brain_activity"] = occupant.get_brain_status()
-		occupantData["pulse"] = occupant.get_pulse(GETPULSE_TOOL)
-		occupantData["blood_o2"] = occupant.get_blood_oxygenation()
-		occupantData["blood_pressure"] = occupant.get_blood_pressure()
-		occupantData["cloneLoss"] = occupant.getCloneLoss()
+		var/displayed_stat = occupant.stat
+		var/blood_oxygenation = occupant.get_blood_oxygenation()
+		if(occupant.status_flags & FAKEDEATH)
```
Comment (FluffyGhoster) [inline code/game/machinery/cryo.dm:139]:
```suggestion
		if(HAS_FLAG(occupant.status_flags, FAKEDEATH))
```

File: code/game/machinery/cryo.dm [hunk 2]
```diff
@@ -76,113 +109,136 @@
...
+			displayed_stat = DEAD
+			blood_oxygenation = min(blood_oxygenation, BLOOD_VOLUME_SURVIVE)
+		var/pulse_result
+		if(occupant.should_have_organ(BP_HEART))
+			var/obj/item/organ/internal/heart/heart = occupant.internal_organs_by_name[BP_HEART]
+			if(!heart)
+				pulse_result = 0
+			else if(BP_IS_ROBOTIC(heart))
+				pulse_result = -2
```
Comment (FluffyGhoster) [inline code/game/machinery/cryo.dm:148]: This looks like a magic number, what is a negative pulse supposed to indicate?

Comment (Generalcamo) [reply code/game/machinery/cryo.dm:148]: No idea, this was copied from the body scanner code

**Review by FluffyGhoster — APPROVED** [review]

**Pull request merged**